packages/common/src/events.js

```diff
@@ -0,0 +1,54 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {isArray} from 'lodash';
+
+/**
+ * Proxies the event binding methods of emitter onto proxy
+ * @param {EventEmitter|EventEmitterProxy} emitter
+ * @param {mixed} proxy (probably a promise)
+ * @returns {EventEmitter} Returns the source emitter to ease use in promise chains
+ */
+export function proxyEvents(emitter, proxy) {
+  [
+    `on`,
+    `once`
+  ].forEach((key) => {
+    proxy[key] = (...args) => {
+      emitter[key](...args);
+      return proxy;
+    };
+  });
+
+  return emitter;
+}
+
+/**
+ * Given a list of events, fires them on drain when they're emitted from source
+ * @param {Array|string} events
+ * @param {EventEmitter} source
+ * @param {EventEmitter} drain
+ * @returns {undefined}
+ */
+export function transferEvents(events, source, drain) {
+  events = isArray(events) || [events];
```

Comment:
yea, you're right. this file is actually something that slipped in early for future share activity support, but i'll fix this here. It should've been a ternary.